infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -634,6 +650,10 @@ sites:
     name: "Rødovre Bibliotek"
     description: "The library site for Rødovre"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOLyDhYdpnunBj8EZNR4fE6lN4vc/+RVVB5n2ei/40xt"
+    primary-domain: www.rdb.dk
```

Comment:
Sure about the order of primary/secondary domains here?

Reply:
yep